core/kubernetes/dev/dev-urls.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+res="$(kubectl get service openhim-mongo-dev-service)"
+
+if [ "$?" == "0" ]; then
+  url="$(minikube service openhim-mongo-dev-service --url)"
```

Comment:
We might need to replace the protocol (`http://`) from this result as the database is accessed via tcp and not http